autoload/ctrlspace/util.vim

```diff
@@ -56,17 +56,24 @@ function! s:handleSwitchbuf(switch) abort
 endfunction
 
 function! s:handleAutochdir(switch) abort
-    if (a:switch ==# "start") && &acd
-        let s:acdWasOn = 1
-        set noacd
-    elseif (a:switch ==# "stop") && exists("s:acdWasOn")
-        set acd
+    if a:switch
+        if !exists("s:acdWasOn")
+            let s:acdWasOn = &autochdir
+        endif
+        set noautochdir
+    elseif exists("s:acdWasOn") && s:acdWasOn
+        set autochdir
         unlet s:acdWasOn
     endif
 endfunction
 
 function! ctrlspace#util#WorkspaceFile() abort
-    return s:internalFilePath("cs_workspaces")
+    if exists('g:ctrlspaceWorkspace')
```

Comment:
This variable seems undocumented. All other use Pascal case; how come this camel case?

Reply:
I added it. You can chenge the name

Reply:
Would you mind using Pascal case and documenting the variable?